content/chapters/software-stack/lab/solution/basic-syscall/arm/hello.s

```diff
@@ -1,6 +1,7 @@
 .section .bss
 
     .lcomm buffer, 128
+    .p2align 12
```

Comment:
This needs a comment stating why it's part of the code: a description of the issue encountered and the QEMU version with the error (it doesn't happen on my side) if `.p2align 12` is absent.